### ISSUE TITLE
Ubuntu 17.10, case sensitive D2K assets

### ISSUE THREAD

**Issue by diddledan** [OpenRA/OpenRA#14319]
I've got a Dune2000 disc from 1998 which the content installer is not recognising. I have run checkdisk.exe which reports:

```plain
/media/dllewellyn/DUNE2000
        Type: CDRom Format: iso
        /media/dllewellyn/DUNE2000/MUSIC/AMBUSH.AUD: (not found)
        /media/dllewellyn/DUNE2000/SETUP/SETUP.Z: (not found)
```

I note that on this system, Ubuntu 17.10, the files and folders in the mount are all lowercase which I'm presuming is why `MUSIC/AMBUSH.AUD` is not matching - on the mountpoint I'm seeing that file as `music/ambush.aud` and likewise for `SETUP/SETUP.Z` and `setup/setup.z`

The disc was mounted automatically via gvfs/gnome with the following signature:

```plain
/dev/sr0 on /media/dllewellyn/DUNE2000 type iso9660 (ro,nosuid,nodev,relatime,nojoliet,check=s,map=n,blocksize=2048,uid=1000,gid=1000,dmode=500,fmode=400,uhelper=udisks2)
```

MD5SUMs are:

```plain
33f43a389029fe68861eba54a036e0c0  /media/dllewellyn/DUNE2000/music/ambush.aud
e24175e7da9754ea2b0d3d301e495668  /media/dllewellyn/DUNE2000/setup/setup.z
```

and SHASUMs are:

```plain
bd5926a56a83bc0e49f96037e1f909014ac0772a  /media/dllewellyn/DUNE2000/music/ambush.aud
b476661e82eeb05949e97fa1f75fed2343174be5  /media/dllewellyn/DUNE2000/setup/setup.z
```

Hopefully that is enough detail :-)

**Comment by penev92** [comment]
From what I see in the mod files, we have setups for installing from two types of CDs with different types of content, and they respectively have:
```
	IDFiles:
		music/ambush.aud: bd5926a56a83bc0e49f96037e1f909014ac0772a
		setup/setup.z: 937f5ceb1236bf3f3d4e43929305ffe5004078e7
```
and
```
	IDFiles:
		music/ambush.aud: bd5926a56a83bc0e49f96037e1f909014ac0772a
		setup/setup.z: 029722e70fb7636f8120028f5c9b6ce81627ff90
```
Is it possible this is a third distribution and that is the cause of the problem?

**Comment by pchote** [comment]
For some reason we forgot to set up d2k with the forced lower case `-linux` variants that work around this problem in the other mods.

**Comment by penev92** [comment]
@diddledan are you able to investigate if/why/how your version of `setup.z` differs from the one described in the installer files, or can you upload it somewhere so we can investigate?

**Comment by diddledan** [comment]
I'm not sure what I'd be looking for, so I've uploaded it to: https://1drv.ms/u/s!AhfazxTKEniEoqYGSEdT_xiVSasEfw (link expires on 12 December 2017)

**Comment by penev92** [comment]
@diddledan I started comparing versions. Turns out there are slight differences in the packaging that was done. Can you also upload the README.TXT file please?

**Comment by diddledan** [comment]
hiya, thanks for looking at it, I've put the readme onto ubuntu's pastebin: https://paste.ubuntu.com/25973945/

**Comment by penev92** [comment]
I will try to get this fixed for the next playtest, which should be in around a month.

**Comment by penev92** [comment]
@diddledan can you please try and drop [mod.log](https://github.com/OpenRA/OpenRA/files/1508229/mod.log) in your `/OpenRA/mods/d2k/` directory and [d2k-e.log](https://github.com/OpenRA/OpenRA/files/1508234/d2k-e.log) in `/OpenRA/mods/d2k/installer/` and renaming them to `*.yaml` (GitHub wouldn't allow me to attach `*.yaml` files).
When you rename `mod.log` to `mod.yaml` you'll replace the current one.
Can you then please verify that the new installer works for you? Just run the game and try to install from your disk.

**Comment by diddledan** [comment]
Those two files successfully allow the game to install off my CDRom. Awesome work! :-)

**Comment by penev92** [comment]
Great!

**Comment by diddledan** [comment]
They work under ubuntu 17.10, too.

**Comment by penev92** [comment]
Thanks. That should make the review process easier ;)

**Comment by reaperrr** [comment]
Closed via #14446.
